src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -142,12 +197,25 @@ export class SearchResults extends Component {
           >
             No schools found for your search criteria.
           </h2>
-          <p>To find participating schools:</p>
-          <ul>
-            <li>Double-check your spelling.</li>
-            <li>Try fewer search terms.</li>
-            <li>Try more general search terms.</li>
-          </ul>
+
+          {showYellowRibbonEnhancements ? (
+            <AlertBox
+              content={TOOL_TIP_CONTENT}
+              headline={TOOL_TIP_LABEL}
+              onCloseAlert={toggleAlertBoxAppearance(false)}
+              isVisible={state.isSearchResultsTipOpen}
+              status="info"
+            />
```

Comment:
Not a necessary change but as a personal pref I might abstract this whole AlertBox into a component ->

```
showYellowRibbonEnhancements  ? (
  <ToolTipAlertBox 
     onCloseAlert={toggleAlertBoxAppearance(false)}
     isVisible={state.isSearchResultsTipOpen}/>
) : // ...
```
If you have a unit test against this component you could check for the existence of a component called `ToolTipAlertBox` so it makes testing a little easier